rgbd/test/test_server.cpp

```diff
@@ -1,37 +1,55 @@
-#include "rgbd/Server.h"
+#include "rgbd/server.h"
 
 #include <ros/init.h>
+#include <ros/names.h>
 #include <ros/rate.h>
 
+#include <image_geometry/pinhole_camera_model.h>
+#include <sensor_msgs/CameraInfo.h>
+#include <sensor_msgs/distortion_models.h>
+
 int main(int argc, char **argv)
 {
     ros::init(argc, argv, "rgbd_transport_test_server");
 
     rgbd::Server server;
-    server.initialize("test", rgbd::RGB_STORAGE_LOSSLESS, rgbd::DEPTH_STORAGE_LOSSLESS);
+    server.initialize(ros::names::resolve("test"), rgbd::RGB_STORAGE_LOSSLESS, rgbd::DEPTH_STORAGE_LOSSLESS);
 
     ros::Rate r(30);
+    cv::Mat rgb_image(480, 640, CV_8UC3, cv::Scalar(0,0,255));
```

Comment:
I guess we should/might/could put this in a helper method, have we used this before? Or isn't it reusable?

Reply:
I think it only used here